include/IFileList.h

```diff
@@ -47,7 +47,9 @@ struct SFileListEntry
 		if (IsDirectory != other.IsDirectory)
 			return false;
 
-		return FullName.equals_ignore_case(other.FullName);
+		const auto& a = FullName.string();
+		const auto& b = other.FullName.string();
+		return std::equal(std::begin(a), std::end(a), std::begin(b), [](auto a, auto b) {return std::tolower(a) == std::tolower(b); });

```

Comment:
I think we have a utility for this somewhere already
https://github.com/Devsh-Graphics-Programming/Nabla/blob/124355408c8a96428205f9decb2c7eb6c75a4073/include/nbl/core/string/stringutil.h#L146

probably bad to redefine the same lambda every time